ddp-workspace/projects/ddp-sdk/src/lib/services/internationalization/languageService.service.ts

```diff
@@ -67,4 +72,11 @@ export class LanguageService {
     const obs = this.changeLanguageObservable(languageCode);
     return obs !== null;
   }
+
+  public getAppLanguageCode(): string {
+    return this.translate.currentLang ||
+      (this.session.session && this.session.session.locale) ||
```

Comment:
I think we shouldn't rely on the session's locale because here is a bug(I'm not sure where: backend, study config, or auth0): if TestBoston user logs in, chooses a language other then English, logs out and then logs in again, session info which stores in the localStorage will contain `locale: en` instead of preferred user's language.